src/SprykerEco/Zed/Payone/Business/Payment/MethodSender/PayoneBankAccountCheckMethodSender.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Payone\Business\Payment\MethodSender;
+
+use Generated\Shared\Transfer\PayoneBankAccountCheckTransfer;
+use Generated\Shared\Transfer\PayoneStandardParameterTransfer;
+use SprykerEco\Shared\Payone\PayoneApiConstants;
+use SprykerEco\Zed\Payone\Business\Api\Adapter\AdapterInterface;
+use SprykerEco\Zed\Payone\Business\Api\Response\Container\BankAccountCheckResponseContainer;
+use SprykerEco\Zed\Payone\Business\Payment\DataMapper\StandartParameterMapperInterface;
+use SprykerEco\Zed\Payone\Business\Payment\PaymentMapperReader;
+
+class PayoneBankAccountCheckMethodSender implements PayoneBankAccountCheckMethodSenderInterface
+{
+    /**
+     * @var \SprykerEco\Zed\Payone\Business\Api\Adapter\AdapterInterface
+     */
+    protected $executionAdapter;
+
+    /**
+     * @var \Generated\Shared\Transfer\PayoneStandardParameterTransfer
+     */
+    protected $standardParameter;
+
+    /**
+     * @var \SprykerEco\Zed\Payone\Business\Payment\DataMapper\StandartParameterMapperInterface
+     */
+    protected $standartParameterMapper;
```

Comment:
Not used.

Reply:
Used in line 68 
`$this->standartParameterMapper->setStandardParameter($requestContainer, $this->standardParameter);`